pytorch_transformers/tokenization_bert.py

```diff
@@ -67,10 +67,10 @@ def load_vocab(vocab_file):
     """Loads a vocabulary file into a dictionary."""
     vocab = collections.OrderedDict()
     with open(vocab_file, "r", encoding="utf-8") as reader:
-        tokens = reader.read().splitlines()
+        tokens = reader.readlines()
     for index, token in enumerate(tokens):
+        token = token[:-1]
```

Comment:
This looks a bit dangerous/unclear to me, can we use something like `token = token.rstrip('\n')` instead?

Reply:
Sure. Thank you!